static/app/views/replays/detail/network/details/onboarding.tsx

```diff
@@ -22,68 +20,6 @@ export const useDismissReqRespBodiesAlert = () => {
   });
 };
 
-export function ReqRespBodiesAlert({
-  isNetworkDetailsSetup,
-}: {
-  isNetworkDetailsSetup: boolean;
-}) {
-  const {dismiss, isDismissed} = useDismissReqRespBodiesAlert();
```

Comment:
Is `useDismissReqRespBodiesAlert` still used anywhere?

Reply:
or maybe this file name could change... it's a curse to export more than one thing imo

Reply:
it is! it's used in `replays/detail/network/details/sections.tsx`